stitch_setup_documentation.md

```diff
@@ -0,0 +1,57 @@
+# FrontApp
```

Comment:
This is duplicate documentation and will likely go out of date. The source of truth for setting up a Front connection in Stitch is here https://www.stitchdata.com/docs/integrations/saas/front

Please remove this file.